internal/controller/variantautoscaling_controller.go

```diff
@@ -110,6 +109,7 @@ var (
 		Version: "v1",
 		Kind:    "ServiceMonitor",
 	}
+	configMapNamespace = os.Getenv("POD_NAMESPACE")
```

Comment:
if value is empty, default to "workload-variant-autoscaler-system"

Reply:
Yeah, thought about this. I wanted to fail. If you need a default, I can add.

Reply:
This will help the units to pass as well. If not has to be set in the test file